changelog.d/20231023_121708_30907815+rjmello_ep_reg_errors.rst

```diff
@@ -0,0 +1,6 @@
+New Functionality
+^^^^^^^^^^^^^^^^^
+
+- When the web service blocks endpoint registration for an expected reason after calling
+  ``globus-compute-endpoint start``, we return a meaningful exit code and print a
+  descriptive message to the terminal.
```

Comment:
We were _already_ doing this, right?  This expands some of the cases where we return a meaningful exit code and more-helpful-than-traceback error message, no?

Reply:
We were, but only for `LOCKED` and `CONFLICT` HTTP responses. I figured mentioning that would add unnecessary details and possible confusion, but I can work it in.